src/overlays/actors/ovl_En_Dno/z_en_dno.c

```diff
@@ -943,8 +943,7 @@ void func_80A73408(EnDno* this, PlayState* play) {
         Cutscene_ActorTranslateAndYaw(&this->actor, play, cueChannel);
     }
 
-    if ((Animation_OnFrame(&this->skelAnime, this->skelAnime.endFrame)) &&
-        (this->animIndex == EN_DNO_ANIM_SHOCK_START)) {
+    if (Animation_OnFrame(&this->skelAnime, this->skelAnime.endFrame) && (this->animIndex == EN_DNO_ANIM_SHOCK_START)) {
         //! FAKE:
         if (1) {}
 
```

Comment:
```suggestion
    if (Animation_OnFrame(&this->skelAnime, this->skelAnime.endFrame)) {
        switch (this->animIndex) {
            case EN_DNO_ANIM_SHOCK_START:
                SubS_ChangeAnimationBySpeedInfo(&this->skelAnime, sAnimations, EN_DNO_ANIM_SHOCK_LOOP, &this->animIndex);
                break;
        }
    }
```
This should get rid of the fake

Reply:
Woah, really nice find!